### DISCUSSION TITLE
Ability to do a headless install using Accessibility tools in the Installer

### DISCUSSION THREAD

**Discussion by TeamLinux01** [elementary/installer#566] [Ideas]
I was thinking it would be nice to be able to completely install Elementary OS without having to relay on a screen or on a very low resolution; I have a Brix computer that has a projector that does 640x480 as its output and a Dell desktop that I use as a server, which doesn't need a monitor attached except when I install an OS.

Separately, it would also be nice to be able to supply an answer file to be able to install without having to input anything additional during the setup.

Update:
#567
#568

**Comment by danirabbit** [comment]
Hey, thanks for opening this discussion! Since this is a mostly a single actionable item and not a super open-ended topic, I think it would make sense to open this as an issue report so that it can be added to the a11y project board.

For the secondary bit about the answer file, it would be great if you could also make sure to file an issue report about that too!

In general, if you have a feature request Issues is the way to go. Discussions is more for things that are a bit more amorphous or like broad overarching initiatives :)

**Reply by TeamLinux01** [reply]
Thanks, I wasn't exactly sure where this should go and just wanted to start something.

I will make sure to keep the idea of using issues for single action items and discussions for more board information.
